app/Livewire/Server/Proxy.php

```
@@ -144,4 +146,135 @@ public function loadProxyConfiguration()
+                if (version_compare($branchNum, $currentBranch, '>')) {
+                    if (! $newestBranch || version_compare($branchNum, $newestBranch, '>')) {
+                        $newestBranch = $branchNum;
+                    }
+                }
+            }
+
+            return $newestBranch ? "v{$newestBranch}" : null;
+        } catch (\Throwable $e) {
+            return null;
+        }
+    }
```

PHPMD (2.15.0)

The method getNewerTraefikBranchAvailableProperty() has a Cyclomatic Complexity of 18. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)